.github/workflows/tests.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
 
             # with --runner=WrapperRunner, it is faster, only ideal with 4 processes (p4)
             # @see https://github.com/rectorphp/rector-src/pull/551#issuecomment-889990905
-            -   run: vendor/bin/paratest -p4 --runner=WrapperRunner ${{ matrix.path }} --colors
+            -   run: vendor/bin/paratest -p5 --runner=WrapperRunner ${{ matrix.path }} --colors
```

Comment:
@samsonasik Just FYI, I don't use paratest at all, so I might be missing something.
This job was constantly failing on this PR, for error 137. AFAIK run out of memory.

I tried to change `-p` to `5` and it seems working :+1: Let me know if there is a better way to handle it.

Reply:
@TomasVotruba thank you, it was slower with `5`, but seems faster now 👍